### PR TITLE
Disable toolbar dropdown if no item in the group is enabled.

### PR DESCRIPTION
**Pull request by martinpovolny**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1633727

### CODE REVIEW

File: src/toolbar/components/toolbar-menu/toolbarListComponent.ts [hunk 1]
```diff
@@ -52,6 +52,12 @@ export class ToolbarListController implements IToolbarListBindings {
       this.toolbarList.items &&
       this.toolbarList.items.filter((item: IToolbarItem) => !item.hidden).length > 0;
   }
+
+  private isToolbarEnabled(): boolean {
+    return this.toolbarList &&
```
Comment (himdel) [inline src/toolbar/components/toolbar-menu/toolbarListComponent.ts]: maybe add ` && this.toolbarList.enabled` to move the whole condition here?

**Comment by JPrause** [comment]
@miq-bot add_label blocker

File: src/toolbar/components/toolbar-menu/toolbarListComponent.ts (hunk 1 above)
Comment (karelhala) [reply src/toolbar/components/toolbar-menu/toolbarListComponent.ts]: Yup, that sounds better, plus use `some` so we don't have to iterate over the whole array.
```TS
private isToolbarEnabled(): boolean {
  return this.toolbarList && 
    this.toolbarList.enabled &&
    this.toolbarList.items &&
    this.toolbarList.items.every((item: IToolbarItem) => item.enabled)
}
```

**Comment by martinpovolny** [comment]
@himdel, @karelhala : please, re-review. Thx!

**Comment by himdel** [comment]
LGTM :)

**Pull request merged**